deliverables/README.md

```diff
@@ -12,16 +12,37 @@ Sign Up for Cloud9 and create a basic Rails project.
 ####Required
 
 What is an "interpreted" language?<br>
-What is IRB?<br>
+An interpreted language is a language that requires an interpreter to be executed  
+
+What is IRB?<br> 
+IRB is a read-eval-print-loop (REPL) for programming in the language Ruby. IRB stands for interactive ruby and can be accessed throgh the Terminal.
+
 What are Objects?<br>
+Everything in Ruby is an object, and every expression in Ruby evaluates to an object  
+
 What are Methods?<br>
+Ruby methods are very similar to functions in any other programming language. Ruby methods are used to bundle one or more repeatable statements into a single unit. Method names should begin with a lowercase letter.
+
 What are Classes?<br>
+Classes act as the factories that build objects. A class in Ruby always starts with the keyword class followed by the name of the class.
+
 What are Blocks?<br>
+A block consists of chunks of code, and you assign a name to a block. The code in the block is always enclosed within braces({}).
+
```

Comment:
A one line block is always enclosed in braces, but in ruby you can also use do and end to open a block, such as:

`10.times do |some_variable|
   puts 'this is a block' 
   puts some_variable
end`

that will print "this is a block" and whatever variable "some_variable" is, 10 times!